graphql_compiler/tests/integration_tests/test_backends_integration.py

```diff
@@ -171,8 +173,21 @@ def test_has_substring_precedence(self, backend_name):
         ]
         self.assertResultsEqual(graphql_query, parameters, backend_name, expected_results)
 
+    @integration_fixtures
+    def test_snapshot_schema_data_from_orientdb_schema(self):
+        schema_data = ordered(generate_schema_data(self.graph_client))
+
+        # To update run:
+        # import json
+        # from ..test_helpers import SCHEMA_DATA_JSON_FILE
+        # with open(SCHEMA_DATA_JSON_FILE, 'w') as outfile:
+        #     json.dump(generate_schema_data(self.graph_client), outfile)
```

Comment:
This seemed like the most convenient place to put this code.

Reply:
This should be added as code and tested as well, otherwise it might drift. Bonus points if you add the pretty-printing step here, so the JSON file is always pretty-printed.